XamlStyler.Mac/Properties/AddinInfo.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using Mono.Addins;
 using Mono.Addins.Description;
 
-[assembly: Addin("XamlStyler", Namespace = "Xavalon", Version = "1.1.5")]
+[assembly: Addin("XamlStyler", Namespace = "Xavalon", Version = "3.2001.0")]
 
 [assembly: AddinName("XAML Styler")]
 [assembly: AddinCategory("IDE extensions")]
```

Comment:
We publish all binaries under "Xavalon" and drop our personal affiliations. Can you update the `AddinAuthor` below to reflect that?

Reply:
Left only Xavalon as AddinAuthor